packages/ui/src/components/IdenticonBadge.tsx

```diff
@@ -44,6 +44,7 @@ export const IdenticonBadge = ({
       color="primary"
       badgeContent={badge}
       anchorOrigin={{ horizontal: 'left', vertical: 'top' }}
+      data-cy={badge === AccountBadge.PURE ? 'badge-pure' : 'badge-multisig'}
```

Comment:
I'd use the badgeContent variable here.
`badge-${badgeContent}`

Reply:
Updated to `data-cy={`badge-${badge}`}` 

(badgeContent property is using the badge prop)